pkgs/misc/drivers/xp-pen-deco-01-v2-driver/default.nix

```diff
@@ -0,0 +1,57 @@
+{ lib, stdenv, fetchzip, libusb, glibc, libGL, xorg, qtx11extras, wrapQtAppsHook, autoPatchelfHook, ... }:
+
+stdenv.mkDerivation {
+  pname = "xp-pen-deco-01-v2-driver";
+  version = "3.2.3.220323-1";
+  src = fetchzip {
+    url = "https://www.xp-pen.com/download/file/id/1936/pid/440/ext/gz.html#.tar.gz";
+    sha256 = "sha256-n/yutkRsjcIRRhB4q1yqEmaa03/1SO8RigJi/ZkfLbk=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    libusb
+    xorg.libX11
+    xorg.libXtst
+    xorg.libXi
+    xorg.libXrandr
+    xorg.libXinerama
+    glibc
+    libGL
+    stdenv.cc.cc.lib
+    qtx11extras
+  ];
+
+  installPhase = let dataDir = "var/lib/xppend1v2"; in ''
+    runHook preInstall
+
+    mkdir -p $out/{opt,bin}
+    cp -r App/usr/lib/pentablet/{pentablet,resource.rcc,conf} $out/opt
+    chmod +x $out/opt/pentablet
+    cp -r App/lib $out/lib
+    sed -i 's#usr/lib/pentablet#${dataDir}#g' $out/opt/pentablet
```

Comment:
Could we use `substituteInPlace` instead? That's a bit more consistent with the rest of nixpkgs, and in my opinion looks cleaner.

Reply:
[`substituteInPlace` dies on null bytes, unfortunately.](https://github.com/NixOS/nixpkgs/blob/42ca9bef09e780eabe84328dd1b730cef978f098/pkgs/stdenv/generic/setup.sh#L751)